examples/dlp_api_example/main.tf

```diff
@@ -110,7 +110,7 @@ EOF
 
 module "dataflow-job" {
   source  = "terraform-google-modules/dataflow/google"
```

Comment:
This would require the root module to be present. I think we have removed the root module. This is the reason lint is failing for this example.